app.rb

```diff
@@ -0,0 +1,125 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+require_relative 'rental'
+require_relative 'book'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_books
+    puts "List of Books (#{@books.length}):"
+    @books.each do |book|
+      puts "#{book.id} - #{book.title} by #{book.author}"
+    end
+  end
```

Comment:
- [ ] Please fix your `list of books` option because when I created a book and after creating a book then I check `list of books` it gives an error and the error is `undefined method `id' for #<Book:0x00007fd57966ce10 @id=89883, @title="my book", @author="book author", @rentals=[]>` so kindly fix it :slightly_smiling_face: 

![Screenshot from 2023-10-04 22-45-58](https://github.com/chudisoft/ruby-school-library/assets/45226848/0e83f867-2b1f-4c15-9615-cf89792eb657)